tests/integrationv2/test_renegotiate_apache.py

```diff
@@ -0,0 +1,138 @@
+import pytest
+import tempfile
+
+from configuration import ALL_TEST_CURVES
+from common import ProviderOptions
+from fixtures import managed_process
+from providers import Provider, S2N
+from utils import invalid_test_parameters, get_parameter_name
+from constants import TEST_CERT_DIRECTORY
+
+from test_renegotiate import TEST_PROTOCOLS, S2N_RENEG_OPTION, S2N_RENEG_ACCEPT
+
+APACHE_SERVER_IP = "127.0.0.1"
+APACHE_SERVER_PORT = 7777
+
+APACHE_SERVER_CERT = TEST_CERT_DIRECTORY + "apache_server_cert.pem"
+APACHE_CLIENT_CERT = TEST_CERT_DIRECTORY + "apache_client_cert.pem"
+APACHE_CLIENT_KEY = TEST_CERT_DIRECTORY + "apache_client_key.pem"
+
+
+def create_get_request(route):
+    return f"GET {route} HTTP/1.1\r\nHost: localhost\r\n\r\n"
+
+
+@pytest.mark.uncollect_if(func=invalid_test_parameters)
+@pytest.mark.parametrize("protocol", TEST_PROTOCOLS, ids=get_parameter_name)
+def test_change_cipher_suite_endpoint_fails_with_no_reneg(managed_process, protocol):
```

Comment:
You probably only need one failure test, parameterized with the route. The loss of the "curve" parameters is pretty unimportant.

Reply:
Added! 40c328c04bd5b7c02beedbbcf7264163987775e7